### PR TITLE
[4.3] Installation: Add hint for handling of existing tables with same prefix

### PR DESCRIPTION
**Pull request by Hackwar**
Pull Request for Issue #18835.

### Summary of Changes
When Joomla is installed in the same database with existing tables with the same prefix as set in the installation, it will rename the existing tables by adding the prefix "bak_". While this was an option in the past, at least in 4.x this has been done silently. This PR adds a hint describing this behavior. 

### Testing Instructions
Do the installation, read the text below the prefix field.

### Actual result BEFORE applying this Pull Request
No text below prefix field.

### Expected result AFTER applying this Pull Request
Text below prefix field. 😛 

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: installation/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -90,6 +90,7 @@ INSTL_DATABASE_NAME_MSG_POSTGRES="The database name is invalid. It must start wi
 INSTL_DATABASE_NO_SCHEMA="No database schema exists for this database type."
 INSTL_DATABASE_PASSWORD_DESC="Either a password you created or a password provided by your host."
 INSTL_DATABASE_PREFIX_DESC="Enter a table prefix or use the randomly generated one."
+INSTL_DATABASE_PREFIX_DUPLICATE_DESC="If you are using an existing database with tables with the same prefix, Joomla will rename those existing tables and add the prefix "bak_" to them."
```
Comment (brianteeman) [inline installation/language/en-GB/joomla.ini]:
```suggestion
INSTL_DATABASE_PREFIX_DUPLICATE_DESC="If you are using an existing database with tables with the same prefix, Joomla will rename those existing tables by adding the prefix "bak_"."
```

File: installation/language/en-GB/joomla.ini [hunk 2]
```diff
@@ -90,6 +90,7 @@ INSTL_DATABASE_NAME_MSG_POSTGRES="The database name is invalid. It must start wi
 INSTL_DATABASE_NO_SCHEMA="No database schema exists for this database type."
 INSTL_DATABASE_PASSWORD_DESC="Either a password you created or a password provided by your host."
 INSTL_DATABASE_PREFIX_DESC="Enter a table prefix or use the randomly generated one."
+INSTL_DATABASE_PREFIX_DUPLICATE_DESC="If you are using an existing database with tables with the same prefix, Joomla will rename those existing tables by adding the prefix "bak_"."
```
Comment (richard67) [inline installation/language/en-GB/joomla.ini]:
```suggestion
INSTL_DATABASE_PREFIX_DUPLICATE_DESC="If you are using an existing database with tables with the same prefix, Joomla will rename those existing tables by adding the prefix \"bak_\"."
```
Shouldn't double quotes inside the string be escaped?

Comment (laoneo) [reply installation/language/en-GB/joomla.ini]: I'm using single quotes in my extension to be on the safe side.

Comment (richard67) [reply installation/language/en-GB/joomla.ini]: Whatever ... but we can't leave it as it is now.

**Review by richard67 — APPROVED** [review]

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 924b77c22c7d2ef0bbcdec35392980789574ae8e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39215">issues.joomla.org/tracker/joomla-cms/39215</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 924b77c22c7d2ef0bbcdec35392980789574ae8e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39215">issues.joomla.org/tracker/joomla-cms/39215</a>.</sub>

**Comment by obuisard** [comment]
Thank you @Hackwar for this PR!

**Comment by zero-24** [comment]
cc @tecpromotion this needs to be coordinated with the TTs as this touches an Installation language string which needs to be merged back into the CMS ideally before the release for as many translations as possible.

**Pull request merged**